src/index.html

```diff
@@ -11,6 +11,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="index.html" 
+      method="post"
+      class="bigform"
+      data-qa="big"
+    >
+  
+      <input 
+        class="big input" 
+        type="search" 
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
+
+    <form 
+      action="index.html"
+      method="post"
+      class="smallform"
+      data-qa="small"
+    >
+
+      <input 
+        class="small input"
```

Comment:
```suggestion
        class="input input--small"
```
better use classes like this